### PR TITLE
PyCon CZ venue / date change announcement

### PR DESCRIPTION
**Pull request by honzajavorek**
Should be published on Tue 25th and synchronized with changes on the PyCon CZ website.

### CODE REVIEW

File: content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+![MeetFactory from inside]({filename}/images/meetfactory6_music_hall_hr.jpg)
+
+**[MeetFactory](http://www.meetfactory.cz/en/)** is a non-profit **international center for contemporary art** founded in 2001 by the artist [David Černý](https://en.wikipedia.org/wiki/David_%C4%8Cern%C3%BD). It is in a very unique, **industrial building** in Prague's quarter Smíchov. It usually hosts music, theatre, gallery and the artist-in-residence program. You can walk through the whole space [on Google Maps](https://www.google.com/maps/@50.053037,14.408284,3a,75y,332h,90t/data=!3m5!1e1!3m3!1s5eyFEmazdNkAAAQfDU22GQ!2e0!3e2!6m1!1e1).
+
+![MeetFactory from outside]({filename}/images/meetfactory1_hr.jpg)
+
+Unfortunately, with change of the venue, also the dates had to be changed. The bad news is, MeetFactory isn't available for us during Sunday 10th of June, because they're hosting the **Prague Museum Night**. For that reason, we decided to move the conference from 9-11 June to 8-10 June. The new expected schedule is:
```
Comment (hroncok) [inline content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: during **Saturday** 10th of June

File: content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+- **Thursday 8th**, MeetFactory - first conference day, talks
+- **Friday 9th**, MeetFactory - second conference day, talks
+- **Saturday 10th**, different venue - workshops & sprints
```
Comment (hroncok) [inline content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: Note about the fact that we have this almost sorted out but don't want to announce the venue before we have it on paper?

File: content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+The good news is, while already being in Prague for PyCon CZ, you can attend the [Prague Museum Night](http://www.prazskamuzejninoc.cz/?l=en)! It is a night during which 80+ Prague's **museums, galleries, and other cultural institutions** are open for free, often with rich accompanying program. Also, you can save Sunday for **travelling home without missing workshops**.
+
+![Prague Museum Night]({filename}/images/2016_124_02.jpg)
+Source: Prague Museum Night's Official Photogallery
+
+We understand this whole thing can cause many inconveniences for you. We worked really hard to resolve the issue as soon as possible, but it still took us significant time. We didn't want to give up the idea about an **extraordinary venue with artistic soul** instead of university lecture rooms.
+
+We may have failed to communicate the uncertainity early enough to prevent you from booking flights or accommodation. On the other hand, originally, we did not expect to shift the dates. If you get into some serious troubles because of the changes, please let us know and we'll try to find a way to help you with re-arranging.
```
Comment (hroncok) [inline content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: Dan Srb: troubles > trouble

File: content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+![we're sorry]({filename}/images/im-sorry.png)
+
+Many of you were asking for **tickets**, but they were still unavailable. As you can now imagine, we couldn't let you buy anything before we had the most important conference details secured. Now when everything is finally sorted, nothing is preventing us to start selling! [Quick, maybe you can still catch an **early bird price**!](https://ti.to/pyvec/pycon-cz-2017)
+
+At this moment we're busy with selecting talks and workshops from the public Call for Papers. Soon we are going to present you with a full program. I can assure you already that there are some amazing pieces waiting for you!
+
+Looking forward to see you <del>soon</del> **in 44 days, in MeetFactory**.
```
Comment (hroncok) [inline content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: Dan Srb: in MeetFactory > at MeetFactory; but I might be wrong

File: content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Modified: 2017-04-24 20:09:00
+Author: Honza Javorek
+Twitter: honzajavorek
+Gravatar: mail@honzajavorek.cz
+About: [Honza](http://honzajavorek.cz) volunteers in the PyCon CZ organization team. His responsibility is to promote the event on social media and to take care of "Public Relations" of the conference.
+
+
+In the past month, the [PyCon CZ conference](https://cz.pycon.org/2017/) **became homeless**. Because of some property issues, the venue we originally announced suddenly became unavailable for any events.
+
+That has put our team into quite unpleasant situation - instead of working hard on the other tasks necessary for the conference to happen, we had to go back to the beginning and start looking for a new place. In a very tight schedule. After several weeks of contacting, visiting, and considering almost 50 venues, we're happy to present you with the replacement: **MeetFactory**!
```
Comment (baradrb) [inline content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: quite an unpleasant
on other tasks
On a very tight

File: content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+![MeetFactory from inside]({filename}/images/meetfactory6_music_hall_hr.jpg)
+
+**[MeetFactory](http://www.meetfactory.cz/en/)** is a non-profit **international center for contemporary art** founded in 2001 by the artist [David Černý](https://en.wikipedia.org/wiki/David_%C4%8Cern%C3%BD). It is in a very unique, **industrial building** in Prague's quarter Smíchov. It usually hosts music, theatre, gallery and the artist-in-residence program. You can walk through the whole space [on Google Maps](https://www.google.com/maps/@50.053037,14.408284,3a,75y,332h,90t/data=!3m5!1e1!3m3!1s5eyFEmazdNkAAAQfDU22GQ!2e0!3e2!6m1!1e1).
```
Comment (baradrb) [inline content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: hosts music events, theatre, gallery exhibitions, and residential artists program.

File: content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md (hunk 1 above)
Comment (baradrb) [inline content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: the date had to change a bit.
on Saturday 10th June,

File: content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md (hunk 2 above)
Comment (baradrb) [reply content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: tak datum workshopů už se neposune, jen nebudeme říkat, kde to bude, dokud nebude papír.

File: content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md (hunk 3 above)
Comment (baradrb) [reply content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: správně

Comment (baradrb) [inline content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: may cause a lot of inconvenience for you.
to help you rearrange.

File: content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md (hunk 4 above)
Comment (baradrb) [reply content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: at

**Review by baradrb — APPROVED** [review]
skvělý text! díky

File: content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md (hunk 2 above)
Comment (honzajavorek) [reply content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: radsi neslibovat, myslim, ze tohle neni tak podstatny a nikoho uz nerozhodi :) ale klidne to tam dam, staci rict!

**Comment by honzajavorek** [comment]
@hroncok @baradrb @benabraham Comments addressed. Do you want to do another round?

@starenka @benabraham When the website is ready, please merge this together with the deploy. The URL of the article is going to be `http://blog.python.cz/pycon-cz-2017-changes-date-and-venue`, link it from the warning on the website, please.

**Review by hroncok — APPROVED** [review]

File: content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+- **Saturday 10th**, different venue - workshops & sprints
+
+The good news is, while already being in Prague for PyCon CZ, you can attend the [Prague Museum Night](http://www.prazskamuzejninoc.cz/?l=en)! It is a night during which 80+ Prague's **museums, galleries, and other cultural institutions** are open for free, often with rich accompanying program. Also, you can save Sunday for **travelling home without missing workshops**.
+
+![Prague Museum Night]({filename}/images/2016_124_02.jpg)
+Source: Prague Museum Night's Official Photogallery
+
+We understand this whole thing may cause a lot of inconvenience for you. We worked really hard to resolve the issue as soon as possible, but it still took us significant time. We didn't want to give up the idea about an **extraordinary venue with artistic soul** instead of university lecture rooms.
+
+We may have failed to communicate the uncertainity early enough to prevent you from booking flights or accommodation. On the other hand, originally, we did not expect to shift the date. If you get into some serious trouble because of the changes, please let us know and we'll try to find a way to help you rearrange.
```
Comment (benabraham) [inline content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md]: @honzajavorek `[let us know](mailto:info@pycon.cz)`

**Pull request merged**